docs/manual/inference.tex

```diff
@@ -229,6 +228,31 @@
 
 You may need to wait a few minutes for the command to complete.
 
+\subsectionAndLabel{Whole-program inference results}{wpi-results}
+
+The result of invoking \<wpi.sh> are log files stored in the \<dljc-out/> folder
+of the target project.
+The \<dljc-out/> folder contains:
+
+\begin{itemize}
+\item \<build\_output.txt>: Logs collected from compiling the target project
+  using the build file found at the top-level directory.
+\item \<javac.json>: The list of all `.java` files for which inference was
+  attempted by \<wpi.sh>, and the options passed to \<javac>.
+\item \<stats.json>: Statistics from the invocation of \<wpi.sh> on the target
+  project, including build time, the number of built \<.jar> files, the number of
+  executable \<.jar> files, the number of \<javac> invocations, and the number of
+  source files.
+\item \<typecheck.out>: The final results of type-checking obtained with the
```

Comment:
Rather than "results", which is vague, use a specific term here. This file actually contains the error messages produced by javac on the last run of WPI, so say that.